cypress/fixtures/analysis.sh

```
@@ -0,0 +1,243 @@
+konveyor.io/target=eap8
+konveyor.io/target=drools
+konveyor.io/target=camel
+konveyor.io/target=hibernate
+konveyor.io/target=jbpm
+)
+
+#
+# Analysis
+#
+printf "\x1DBEGIN-MAIN\x1D\n" > ${file}
+echo -n "---
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)